settings.gradle

```diff
@@ -131,15 +138,15 @@ if (sparkVersions.contains("3.1")) {
 }
 
 if (sparkVersions.contains("3.2")) {
-  include ':iceberg-spark:spark-3.2_2.12'
-  include ':iceberg-spark:spark-extensions-3.2_2.12'
-  include ':iceberg-spark:spark-runtime-3.2_2.12'
-  project(':iceberg-spark:spark-3.2_2.12').projectDir = file('spark/v3.2/spark')
-  project(':iceberg-spark:spark-3.2_2.12').name = 'iceberg-spark-3.2_2.12'
-  project(':iceberg-spark:spark-extensions-3.2_2.12').projectDir = file('spark/v3.2/spark-extensions')
-  project(':iceberg-spark:spark-extensions-3.2_2.12').name = 'iceberg-spark-extensions-3.2_2.12'
-  project(':iceberg-spark:spark-runtime-3.2_2.12').projectDir = file('spark/v3.2/spark-runtime')
-  project(':iceberg-spark:spark-runtime-3.2_2.12').name = 'iceberg-spark-runtime-3.2_2.12'
+  include ":iceberg-spark:spark-3.2_${scalaVersion}"
```

Comment:
It looks like we should never set `scalaVersion` to "" because it will cause build failures. Just leaving it as 2.12 unless we override seems reasonable to me.